src/components/user-card.yml

```diff
@@ -0,0 +1,3 @@
+profileButton:
+  en: view profile
```

Comment:
Should be uppercased: "**V**iew profile" (like "Enter room", "Accept invitation", etc.)

Reply:
Oh, corrected, thanks!